### PR TITLE
SET-403 create 6.4 pipe

### PR DESCRIPTION
**Pull request by baranowb**
Issue: https://issues.redhat.com/browse/SET-403

### CODE REVIEW

File: cryo-pipeline [hunk 1]
```diff
@@ -16,6 +16,9 @@ def dumpParameters(String outputFile) {
     println line
     lines = lines + line
   }
+  lines = lines + "export HARMONIA_SCRIPT=\"${env.HARMONIA_SCRIPT}\"\n"
```
Comment (gaol) [inline cryo-pipeline:19]: We don't need to export the `HARMONIA_SCRIPT` IMO.

**Comment by gaol** [comment]
LGTM

**Comment by rpelisse** [comment]
LGTM
@spyrkob any comments?

File: cryo-pipeline (hunk 1 above)
Comment (baranowb) [reply cryo-pipeline:19]: yeah, we do.

**Comment by spyrkob** [comment]
LGTM

File: cryo-pipeline (hunk 1 above)
Comment (gaol) [reply cryo-pipeline:19]: OK, please forget my comment, this is good to me.

**Pull request merged**